error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/ClassCastLambdaUsage.java

```diff
@@ -48,7 +49,10 @@ public Description matchLambdaExpression(LambdaExpressionTree tree, VisitorState
     }
 
     Type type = ASTHelpers.getType(typeCast);
-    if (type == null || type.isParameterized() || type.isPrimitive()) {
+    if (type == null
```

Comment:
I'm confused now, was the pitest warning already here or is it new 🤔?

Reply:
AFAIK:

1. It was in the original commit, pre suggestions.
2. It was introduced with this [PR](https://github.com/PicnicSupermarket/error-prone-support/pull/1385/files), so it is probably referencing mutating the `#isPrimitive` call 🤔

Reply:
Thanks for checking @mohamedsamehsalah 🚀🚀🚀!

Reply:
Any clue why is this reported? 😅

Reply:
> Any clue why is this reported? 😅

I re-read the warning line
> A change can be made to line 52 without causing a test to fail

which seems to come from mutating the nullability check; which seems common for similar PRs (for example, [here](https://github.com/PicnicSupermarket/error-prone-support/pull/494/files)).